scanner/Makefile

```diff
@@ -69,6 +69,9 @@ build-d   := bin
 
 build-t      := $(addprefix $(build-d)/,$(notdir $(wildcard cmd/*)))
 build-deps-t := deps
+ifdef NODEPS
+	build-deps-t :=
+endif
```

Comment:
Not sure if it's a great idea to use an empty variable as a target in practice. Makefiles are designed to work with well-defined phony targets or file names. It seems this is not against the rules, but it's not a common practice and I am not sure if there are unintended consequences.

Reply:
I updated it to gate using `build-deps-t` as a dependency instead of gating the target name